usecases/security/enforce_security_testrun.go

```diff
@@ -0,0 +1,23 @@
+package security
+
+import (
+	"github.com/checkmarble/marble-backend/models"
+	"github.com/cockroachdb/errors"
+)
+
+type EnforceSecurityTestRun interface {
+	EnforceSecurity
+	CreateTestRun(organizationId string) error
+}
+
+type EnforceSecurotyTestRunImpl struct {
+	EnforceSecurity
+	Credentials models.Credentials
+}
+
+func (e *EnforceSecurotyTestRunImpl) CreateTestRun(organizationId string) error {
+	return errors.Join(
+		e.Permissions(models.BUILDER_PERMISSIONS),
```

Comment:
Ok this works, but I'm wondering if it's necessary 🤔 
I might have misled you by saying "you need to be builder to have the permission". Perhaps just something like `e.Permission(models.SCENARIOS_CREATE),`  would suffice (which is contained in the builder role)